src/styles/blocks/catalog.scss

```
@@ -21,5 +21,12 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition: color 0.3s, transform 0.3s;
```

create a variable for transition

```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```